### ISSUE TITLE
Using template as specified does not seem to work

### ISSUE THREAD

**Issue by kjprice** [remix-run/examples#267]
tl;dr can we please update [the readme](https://github.com/remix-run/examples/blob/e2f3e70a367d2057dcfd65724a24a823ba524077/README.md) to have template instructions that will work for an impatient noob?

Hello!

I'm new to remix and love it!

The examples are great but I had a tough time using one of the examples as a template.

## Problem

In [the readme](https://github.com/remix-run/examples/blob/e2f3e70a367d2057dcfd65724a24a823ba524077/README.md) I followed the instructions:

> You can also initialize a new project with any of these examples using the --template flag of the create-remix CLI.

And used the example `npx create-remix@latest --template examples/<example-dir>`

Running this on my own, to use the [socket.io example](https://github.com/remix-run/examples/tree/e2f3e70a367d2057dcfd65724a24a823ba524077/socket.io), I ran the following command:

```
npx create-remix@latest --template examples/socket.io
```

However, I received the following error:

```
🚨 Invalid template selected. Please try again.
```

## Solution

After reviewing the [docs related to templates](https://github.com/remix-run/remix/blob/29329819c62208553861cc11982f904cf1b5e79e/docs/other-api/dev.md#remix-create---template) I experimented with a few possibilities.

These did not work for me:

```
npx create-remix@latest --template examples/main/socket.io
npx create-remix@latest --template remix-run/examples/blob/main/socket.io
npx create-remix@latest --template remix-run/examples/socket.io
```

Finally, using the full github path, this did work for me:

```
npx create-remix@latest --template https://github.com/remix-run/examples/tree/main/socket.io
```

My solution I believe is overly verbose than necessary, but if we can update the readme to have an example that will definitely run that would be great.

Thanks again for an amazing product!

**Comment by lasas** [comment]
> I guess this is because of the `.` in the folder name as doing `npx create-remix@latest --template examples/tailwindcss` just works as expected

Well, I tried it and it doesn't work for me, errors with: _Oh no! There was a problem fetching the file from GitHub. The request responded with a 404 status. Please try again later._

But workaround using full repository name worked: `npx create-remix@latest --template https://github.com/remix-run/examples/tree/main/tailwindcss`.

**Comment by SivertGullbergHansen** [comment]
> Well, I tried it and it doesn't work for me, errors with: _Oh no! There was a problem fetching the file from GitHub. The request responded with a 404 status. Please try again later._
> 
> But workaround using full repository name worked: `npx create-remix@latest --template https://github.com/remix-run/examples/tree/main/tailwindcss`.

I experience the same thing. Seems it has not been fixed yet.

**Comment by maxrchung** [comment]
> Well, I tried it and it doesn't work for me, errors with: Oh no! There was a problem fetching the file from GitHub. The request responded with a 404 status. Please try again later.

For this error, try adding `remix-run/` to the front of the template parameter. [The docs](https://remix.run/docs/en/main/other-api/create-remix#create-remix---template) say to add `remix-run/` but [not the README](https://github.com/remix-run/examples/blob/ccc82c399d22f89d8925a3aa671238242d4b1fbb/README.md?plain=1#L16). I opened a PR to update the README: https://github.com/remix-run/examples/pull/435

As for the original `socket.io` having a period issue, I just did a quick test and think it's been fixed by https://github.com/remix-run/remix/pull/7277. Here's some logs for my successful create:

```
PS C:\Users\Max\Desktop> npx create-remix@latest --template remix-run/examples/socket.io

 remix   v2.8.0 💿 Let's build a better website...

   dir   Where should we create your new project?
         ./my-remix-app

      ◼  Template: Using remix-run/examples/socket.io...
      ✔  Template copied

   git   Initialize a new git repository?
         No

  deps   Install dependencies with npm?
         No
      ◼  Skipping install step. Remember to install dependencies after setup with npm install.

  done   That's it!

         Enter your project directory using cd .\my-remix-app
         Check out README.md for development and deploy instructions.

         Join the community at https://rmx.as/discord
```

**Comment by machour** [comment]
Fixed in #435

**Comment by kunalpal97** [comment]
C:\Desktop 1\Protega AI\protega-ai-shopify-app>npx create-remix@latest --template remix-run/examples/socket.io

 remix   v2.16.6 💿 Let's build a better website...

   dir   Where should we create your new project?
         ./my-remix-app

      ◼  Template: Using remix-run/examples/socket.io...
      ✔  Template copied

   git   Initialize a new git repository?
         No

  deps   Install dependencies with npm?
         Yes

      ✔  Dependencies installed

  done   That's it!

         Enter your project directory using cd .\my-remix-app
         Check out README.md for development and deploy instructions.

         Join the community at https://rmx.as/discord


**Guys for this this Works as of now so you can try this**
